charts/alfresco-repository/docs/keystores.md

```diff
@@ -0,0 +1,83 @@
+# Using custom keystore
+
+Alfresco uses a set of keystores for different purposes. Among others thing
+keystores can be used for metadata encryption or traffic encryption.
+
+The alfresco-content-repository image comes with a keystore which contains a
+single security key. That key is primarily used for metadata encryption and it
+required when the repository starts up. In production environment it is
+recommended to use a custom keystore rather than the one present in the image.
+
+One may also need to have additional keystores deployed, for instance to allow
+using TLS encryption for a service the repository provides using an x509
```

Comment:
TLS >> mTLS

Reply:
I'm not mentioning mTLS here on purpose as it is very likely that the other end TLS session might not have the same support for keystore. For now this is limited to repository